includes/parser/Tidy.php

```diff
@@ -42,7 +42,7 @@ public function getWrapped( $text ) {
 		$this->mMarkerIndex = 0;
 
 		$wrappedtext = preg_replace_callback( ParserOutput::EDITSECTION_REGEX,
-			array( &$this, 'replaceEditSectionLinksCallback' ), $text );
+			array( $this, 'replaceEditSectionLinksCallback' ), $text );
 
 		$wrappedtext = '<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"'.
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "$wrappedtext". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)